assets/css/style.css

```diff
@@ -32,8 +41,40 @@
   align-items: center;
 }
 
+.toggle {
+  display: block;
+}
+
```

Comment:
- [ ]  Kyndly make sure you stick to the figma template as much as posible. Please use the correct font size and font family 

## your design

![image](https://user-images.githubusercontent.com/95243996/201778937-c452a12f-abc6-4288-9366-89114c282d6c.png)

## Figma design 

![image](https://user-images.githubusercontent.com/95243996/201779121-6856d0a8-5d5b-4706-a160-669d2ab73d12.png)